documentation/src/docs/recovery-procedures/enable-durable-data-log.md

```diff
@@ -0,0 +1,88 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Enable Durable Data Log
+
+* [Issue](#issue)
+* [Repair Procedure](#repair-procedure)
+
+# Issue
+A `DurableDataLog` is the abstraction that the Segment Store uses to interface with the specifics of
+writing to Tier-1 (i.e., Bookkeeper). As visible in that interface, a `DurableDataLog` can be _enabled_ 
+(writable) or _disabled_ (non-writable). In general, when a log becomes _disabled_, it means that there 
+has been an error for which the Segment Store thinks it is safer to stop writing to the log. A typical
+message indicating this situation is as follows:
+```
+2022-11-03 06:14:07,826 4991892 [core-11] ERROR i.p.s.server.logs.DurableLog - DurableLog[18] Recovery FAILED.
+io.pravega.segmentstore.storage.DataLogDisabledException: BookKeeperLog is disabled. Cannot initialize.
+```
+
+We can classify errors that can lead to disable a `DurableDataLog` into _persistent_ and _transient_.
+Persistent errors are those that impact the data and/or metadata of the log (i.e., data corruption).
+Recovering from these errors may require data recovery procedures as explained in other documents of this
+section. Attempts to enable a log in this state are not advisable and will likely result in the Segment
+Store disabling the log again, once the data corruption is detected. On the other hand, Transient errors 
+are severe but recoverable errors that might lead to also disable the log, such an out of memory problem. 
+This article focuses on the latter category and describes how to re-enable a `DurableDataLog` that has been 
+_disabled_ for reasons different from data corruption.
+
+# Repair Procedure
+
+1. First, configure the [Pravega Admin CLI](https://github.com/pravega/pravega/blob/master/cli/admin/README.md)
+from a location/instance that can access the Zookeeper and Bookkeeper services deployed in your cluster.
+
+2. With the Pravega Admin CLI in place, run the following command: 
+```
+bk list
+...
+{
+  "key": "log_summary",
+  "value": {
+    "logId": 1,
+    "epoch": 81846,
+    "version": 1062379,
+    "enabled": false,
+    "ledgers": 4,
+    "truncation": "Sequence = 344181499234420, LedgerId = 3781242, EntryId = 2164"
+  }
+}
+...
+```
+This command lists all the available Bookkeeper Logs in this cluster (please, be sure that the Pravega
+Admin CLI has the `pravegaservice.container.count` parameter set to the same number of Segment Containers 
+as in the cluster itself). The output of the above command shows that all the disabled Bookkeeper logs
+exhibit `"enabled": false`.
+
+3. Next, we need to be sure that we can recover the disabled Bookkeeper log(s) as a way to verify that
+there are no data corruption issues. To this end, you need to run the following Pravega Admin CLI command
+on all the disabled logs:
+```
+container recover [ID_OF_DISABLED_CONTAINER]
```

Comment:
This allows to check whether container can recover from Persistent errors, but if BookKeeper log been disabled due to transient errors (like OOM), then should we provide some way to detect such errors before enabling Bookkeeper log?

Reply:
This is the point of running `container recover`: if it works, data and metadata is good, so we can safely enable the container. If it does not work, then there is a data corruption issue that should be handled differently.